airsonic-main/src/main/java/org/airsonic/player/service/MediaScannerService.java

```diff
@@ -49,7 +54,9 @@
     private MediaLibraryStatistics statistics;
 
     private boolean scanning;
-    private Timer timer;
+    
+    private ScheduledExecutorService scheduler;
```

Comment:
It seems that you mixed this PR with #1376 :)

Reply:
No, this isn't a mixup.

This is how GitHub deals with commits on successive PRs built on each other (please compare commit hashes for this PR and https://github.com/airsonic/airsonic/pull/1376 ). This PR is a successor of https://github.com/airsonic/airsonic/pull/1376 and built on top of it.

Once you merge https://github.com/airsonic/airsonic/pull/1376, the effective change in this PR is the remaining commit hashes that it doesn't share with https://github.com/airsonic/airsonic/pull/1376 (i.e. everything excluding https://github.com/airsonic/airsonic/pull/1377/commits/87285668606fa3ee6be23aecd7be012e59aeea8d which is part of https://github.com/airsonic/airsonic/pull/1376)

This is also the reason you shouldn't change commit hashes on PRs when merging them on master such as what happened in https://github.com/airsonic/airsonic/pull/1357 and actually even a recently merged one here https://github.com/airsonic/airsonic/pull/1353. It'll result in immediate conflicts in subsequent PRs built on top of another PRs because Git will find no commit history for the hashes in one PR, and immediately declare a merge conflict for all subsequent PRs built on the merged PR that has had its commit history changed. Additionally , the local branches won't recognize they've been merged and refuse to delete unless you force delete them (-D)